### PR TITLE
Version bump to 2.2.1

### LINKED ISSUE #4: Optional Overlay
How to skip the overlay as i dont need it

### LINKED ISSUE #34: No call screen option
Whenever there is an incoming call, I can only choose between answer and decline.

OnePlus 5T 9.0 Pie
December Update
Latest Phone version from APKmirror

### PR DESCRIPTION
**Pull request by thehappydinoa**
- Closes #4
- Fixes keycheck
- Updated versionCode for testing

### CODE REVIEW

File: common/install.sh [hunk 1]
```diff
@@ -71,64 +80,28 @@ chooseportold() {
...
+$FUNCTION "UP"
+ui_print "   Press Vol Down:"
+$FUNCTION "DOWN"
+# fi
 
-# backup
-if [ -f /data/data/com.google.android.apps.nexuslauncher/databases/launcher.db ]; then
+ui_print " "
+ui_print " - Select Option -"
+ui_print "   Do you want overlays enabled:"
```
Comment (ThatNerdyPikachu) [inline common/install.sh]: Can you tell the user what the overlays allow?

File: common/install.sh [hunk 2]
```diff
@@ -71,64 +80,28 @@ chooseportold() {
...
-    if $FUNCTION; then
-      ui_print " "
-      ui_print "   Did not restore!"
-    else
-      rm -f /data/media/0/.launcher.db.backup
-      ui_print " "
-      ui_print "   Did not restore (but ERASED backup)!"
-  fi
+  ui_print "   Disabling overlays..."
+  sed -i -e 's/ro.boot.vendor.overlay.theme/#ro.boot.vendor.overlay.theme/g' $INSTALLER/common/system.prop
```
Comment (ThatNerdyPikachu) [inline common/install.sh]: Can we go with "normal comment style" for this? (``# sed ...``)

File: common/post-fs-data.sh [hunk 3]
```diff
@@ -3,9 +3,14 @@
 # This will make your scripts compatible even if Magisk change its mount point in the future
 MODDIR=${0%/*}
 
-if [[ $(pm list packages "com.google.android.apps.wellbeing") ]]; then
+if [ $(getprop ro.build.version.sdk) -eq 28 ]; then
+  # If PIE
```
Comment (ThatNerdyPikachu) [inline common/post-fs-data.sh]: ``# If PIE`` -> ``# If Pie`` (Btw, where did you get the SDK version from? Why is this needed only on Pie?)

**Review by ThatNerdyPikachu — DISMISSED** [review]

File: module.prop [hunk 4]
```diff
@@ -1,7 +1,7 @@
 id=Pix3lify
 name=Pix3lify
-version=v2.2
-versionCode=220
+version=v2.2.11
+versionCode=2211
```
Comment (ThatNerdyPikachu) [inline module.prop]: We can really just increment ``v2.2.1`` to ``v2.2.2``, as specified by semver

Comment (thehappydinoa) [reply module.prop]: Yes, but if we do testing we have to bump

Comment (thehappydinoa) [reply module.prop]: Actually you are correct

Comment (ThatNerdyPikachu) [reply module.prop]: :)

**Comment by ThatNerdyPikachu** [comment]
@thehappydinoa please make sure i fixed everything before merging

**Review by thehappydinoa — COMMENTED** [review]
LGTM

**Review by ThatNerdyPikachu — APPROVED** [review]
